deploy/src/main/deploy/openshift_s2i/hono-template.yml

```diff
@@ -407,7 +407,7 @@ objects:
           name: fabric8-s2i-java-custom:2.3
         env:
         - name: MAVEN_ARGS_APPEND
-          value: -B -pl org.eclipse.hono:hono-adapter-amqp-vertx --also-make -Pnetty-tcnative
+          value: -B -pl org.eclipse.hono:hono-adapter-amqp-vertx --also-make -Pnetty-tcnative -Dmaven.test.skip=true
```

Comment:
hmm, is that really what you want to do, i.e. **not** run the tests?

Reply:
That is what I need to do, in order to work around this issue.

On the other side, this is a released version of Hono. So I am not sure if re-running the unit tests, for every local installation, is necessary.

Reply:
I see. Just wanted be sure we know what we're doing ;-)